index.html

```diff
@@ -18,12 +18,12 @@
     <header>
```

Comment:
- [ ] Kindly make your title more descriptive eg you can use `Davian full-stack webdeveloper`.